src/creature.h

```diff
@@ -91,14 +91,22 @@ class Creature
         /**
          * Simplified attitude towards any creature:
          * hostile - hate, want to kill, etc.
-         * friendly - avoid harming it, maybe even help.
          * neutral - anything between.
+         * friendly - avoid harming it, maybe even help.
+         * any - any of the above, used in safemode_ui
          */
-        enum Attitude {
+        enum Attitude : int {
             A_HOSTILE,
             A_NEUTRAL,
             A_FRIENDLY,
+            A_ANY
```

Comment:
Well, it is a valid return value of attitude_to, it's just not used.
It would take quite the rewrite to remove it from the list again.